### PR TITLE
Unified user notifications

### PR DESCRIPTION
**Pull request by MyIgel**
Changed all user notifications to be saved in the session using the same structure and output globally

### CODE REVIEW

File: src/Controllers/NotificationType.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
```
Comment (ximex) [inline src/Controllers/NotificationType.php:2]: add `declare(strict_types=1);` in new files

File: src/Renderer/Twig/Extensions/Notification.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<?php
+
```
Comment (ximex) [inline src/Renderer/Twig/Extensions/Notification.php:2]: add `declare(strict_types=1);` in new files

File: tests/Unit/Renderer/Twig/Extensions/NotificationTest.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
+<?php
+
```
Comment (ximex) [inline tests/Unit/Renderer/Twig/Extensions/NotificationTest.php:2]: add `declare(strict_types=1);` in new files

**Review by ximex — CHANGES_REQUESTED** [review]

File: src/Controllers/NotificationType.php (hunk 1 above)
Comment (MyIgel) [reply src/Controllers/NotificationType.php:2]: https://github.com/engelsystem/engelsystem/pull/1055 is not merged so :shrug:

**Review by weeman1337 — APPROVED** [review]
Looks good :+1: 

Update: Looks not good :D

**Review by weeman1337 — CHANGES_REQUESTED** [review]
Breaks at least add user to shift notification

image

**Comment by MyIgel** [comment]
Eventually having #939 merged first and then adding the real user name to notifications might be useful here?

**Review by weeman1337 — APPROVED** [review]
LGTM :+1:

**Pull request merged**